protos/mocap/mocap.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+package mavsdk.rpc.mocap;
+
+option java_package = "io.mavsdk.mocap";
+option java_outer_classname = "MocapProto";
+
+/**
+ * TODO
+ */
+service MocapService {
+    /*
+     * TODO
+     */
+    rpc CreateVisionPositionEstimateStream(stream VisionPositionEstimateRequest) returns(stream VisionPositionEstimateResponse) {}
+    rpc CreateAttPosMocapStream(stream AttPosMocapRequest) returns(stream AttPosMocapResponse) {}
+}
+
+message VisionPositionEstimateRequest {
+    VisionPositionEstimate vision_position_estimate = 1;
+}
+message VisionPositionEstimateResponse {
+  MocapResult mocap_result = 1;
+}
+
+// Global position/attitude estimate from a vision source.
+message VisionPositionEstimate {
+    uint64 time_usec = 1; // Position frame timestamp UNIX Epoch time (0 to use Backend timestamp)
```

Comment:
This can be sensible for FMU:
[time delay](https://dev.px4.io/v1.9.0/en/ros/external_position_estimation.html#tuning-EKF2_EV_DELAY)